src/platforms/android/configuration/integrations/apollo3.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Apollo3
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.java.apollo3
+description: "Learn more about the Sentry Apollo3 integration for the Android SDK."
+categories:
+  - mobile
+redirect_from:
+  - /platforms/android/performance/instrumentation/apollo3/
```

Comment:
You shouldn't need this redirect since this is an entirely new page unless there's somewhere you're linking from that's generating this link

```suggestion
```